src/item/views/modals/AddFragmentToCollection.tsx

```diff
@@ -173,6 +179,9 @@ export const AddFragmentToCollection: FunctionComponent<AddFragmentToCollectionP
 	};
 
 	const renderAddFragmentToCollectionModal = (collections: { id: number; title: string }[]) => {
+		const initFlowPlayer = () =>
+			!playerToken && fetchPlayerToken(externalId).then(data => setPlayerToken(data));
```

Comment:
Update player-service to not catch and log the error, but instead just let the function throw the error
then add a catch here and show an error toast